packages/amplify_api_rest/example/README.md

```diff
@@ -0,0 +1,16 @@
+# amplify_api_rest_example
```

Comment:
Make consistent with other example apps.

Reply:
Thanks our merged example app has this change.